_posts/2016-02-09-meetup.md

```diff
@@ -0,0 +1,77 @@
+---
+title: Bringing Open-Source Practices to Your Day Job
+author: aaron
+layout: post
+categories:
+- posts
+- meetups
+when: 2016-02-16T19:30:00-06:00
+---
+
+{% assign twiturl = 'https://twitter.com/BenjaminCoe' %}
+{% assign sponsurl = 'http://www.ibm.com/design/' %}
+
+<div class="ajs-box">
+  We're not in our usual location! We'll be meeting at Spredfast this month in the Silicon Labs building, so please plan accordingly.
```

Comment:
Yo! I'd clarify a slightly more, because there's 2 Silicon Labs buildings pretty much a block from each other. Perhaps:

---

We'll be meeting at Spredfast this month in the Silicon Labs Building on Colorado/Cesar Chavez ([map](https://www.google.com/maps/place/200+W+Cesar+Chavez+St,+Austin,+TX+78701/@30.2642656,-97.7470567,18z/data=!3m1!4b1!4m2!3m1!1s0x8644b50602c5b57d:0x4c4d44de892b1d04)).